integreat_cms/cms/forms/languages/language_form.py

```diff
@@ -54,6 +56,29 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.fields["primary_country_code"].choices = sorted_language_choices
         self.fields["secondary_country_code"].choices = sorted_language_choices
 
+        # Make the language color field required
+        self.fields["language_color"].required = True
```

Comment:
Thank you for updates 👍 Yes, we have to adjust the tests. `language_color` is a new field and therefore not correctly handled in the test, that's the cause (I guess).

Reply:
@MizukiTemma yep, that is what is saying, but I cannot identify the file where I should add that field :(

Reply:
I suggest to remove `blank=True` and `null=True` from the `language_color` in the model to allow no language to exist without color, before adjusting the tests.
But then we have to assign a color to each existing language in the migration file (otherwise it goes broken since languages stored in the database don't have any color so far), and the design for the color-language pairs can be now changed, because we asked for color/design for total access numbers too.

So, let's wait 😉

Reply:
@MizukiTemma ahh I see now. I tried filling the necessary objects in test_data.json without adding it as well to view_config.py and that way I was also getting errors. Now I see why, thank you a lot :)